### ISSUE TITLE
add an option for printing the outputs of configure-commands, build-commands etc

### ISSUE THREAD

**Issue by devcurmudgeon** [devcurmudgeon/ybd#222]
23:11 < jjardon> paulsherwood: what is the way to tell ybd: show me all the configure and build output to the terminal
Day changed to 27 May 2016
06:49 < paulsherwood> jjardon: there is no option for that, currently
06:52 < jjardon> paulsherwood: I think it would be useful; that option would have save me a couple of hours yesterday 
06:55 < paulsherwood> jjardon: ok. you mean you want all output from configure-commands and build-commands visible, along with what it currently logs?
06:56 < jjardon> Yeah; basically I needed to know what was the output of the ./configure command to check if a chunk was correctly configured 
06:57 < jjardon> Maybe a -v/--verbose option?
08:29 < gtristan> jjardon, you know it goes into the log of the given build already right ?
08:31 < gtristan> jjardon, I think that kind of option can be awkward to try to use, especially when you have > 1 instances (and are building many chunks in parallel)... directing all stdout/stderr 
                  from many builds to the console at once is a bit weird :-S
10:07 -!- mode/#baserock [+v ssam2] by ChanServ
10:45 < jjardon> gtristan: yep, but in this case the build is done in a remote server where I only have access to the console output
10:48 < radiofree> Set the log file location to some remotely accessible location so you can read it via a web browser or something?
10:51 < jjardon> I think having the option in ybd is way more convenient 
10:52 < jjardon> but yeah, thats an option I will use next time if there is not other way: what is the option to set the log file?
10:53 < SotK> iirc you have to use tee
10:53 < radiofree> It would probably be easier to just share the artifact dir over http
10:54 < edcragg> that would be quite handy
10:56 <+pedroalvarez> I assume the goal here is to make clear what is going on, when making builds on a remote CI server
10:58 <+pedroalvarez> I don't think there is a perfect solution, although IMO is nice to have a --build-log-in-stdout option, even if that is mixed, but maybe prefixing with the chunk name
